powershell-adapter/copy_files.txt

```diff
@@ -1 +1,8 @@
 powershell.resource.ps1
+\psDscAdapter\psDscAdapter.psd1
```

Comment:
I don't think the leading slash is needed, makes it seem like you are copying from root?

Reply:
fixed